src/main/java/net/minecraftforge/client/ClientRegistry.java

```diff
@@ -32,6 +34,7 @@
 public class ClientRegistry
 {
     private static Map<Class<? extends Entity>, ResourceLocation> entityShaderMap = new ConcurrentHashMap<>();
+    private static Map<UseAnim, HumanoidModel.ArmPose> useAnimationMap = new ConcurrentHashMap<>();
```

Comment:
Should be final

Reply:
Should I make the entityShaderMap final aswell?

Reply:
No, but this comment is moot anyways if you apply the changes in my last comment